app/src/main/java/com/jerboa/ui/components/inbox/InboxActivity.kt

```diff
@@ -407,7 +407,8 @@ fun InboxTabs(
                                                 }
                                             },
                                             onPostClick = { postId ->
-                                                appState.toPost(id = postId)
+                                                goToComment(commentReplyView)
+                                                markAsRead(commentReplyView)
```

Comment:
This will cause it to mark already marked as read comments as not read yet.
Do the same as
```kt
                              // Do not mark already read reply as read
                                                if (!crv.comment_reply.read) {
                                                    markAsRead(crv)
                                                }
```

Reply:
Also if that `postId` isnt used then refactor that function to not require it